### PR TITLE
layout user and list category

### PR DESCRIPTION
**Pull request by at-betruong**
## Description
- Layout user
- Show api category user
## Reference Requirements
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
YES | NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: resources/views/public/includes/head.blade.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<meta charset="utf-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+<title>MOBILE-SHOP</title>
```
Comment (at-xinhnguyen) [inline resources/views/public/includes/head.blade.php]: trans đi em

File: resources/views/public/includes/header.blade.php [hunk 2]
```diff
@@ -0,0 +1,136 @@
+<!-- HEADER -->
+<header>
+  <!-- top Header -->
+  <div id="top-header">
+    <div class="container">
+      <div class="pull-left">
+        <span>Welcome to Mobile-shop!</span>
```
Comment (at-xinhnguyen) [inline resources/views/public/includes/header.blade.php]: trans này luôn em

File: app/Traits/ApiResponser.php [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+     * @return Illuminate\Support\Collection
+     */
+    protected function paginate(Collection $collection)
+    {
+        $rules = [
+            'perpage' => 'integer|min:2|max:50'
+        ];
+        Validator::validate(request()->all(), $rules);
+        $page = LengthAwarePaginator::resolveCurrentPage();
+        $prePage = config('define.limit_rows');
```
Comment (at-xinhnguyen) [inline app/Traits/ApiResponser.php]: này phải là `perPage` chứ em

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**